README.rst

```diff
@@ -99,6 +97,41 @@ check out our `docs <https://simplepath.readthedocs.org>`_.
         'to': 'people',
     }
 
+Here is another example if your data is stored in an object.
```

Comment:
@gregarmer do you think this should go in readme? I internally debating whether this should be advertised in readme since this is not a core feature of simplepath? your thoughts would be appreciated